Source/CarthageKitTests/ResolverSpec.swift

```diff
@@ -53,12 +53,13 @@ class ResolverSpec: QuickSpec {
 			let testCartfile: Cartfile = self.loadTestCartfile("TestCartfile")
 			let producer = resolver.resolveDependenciesInCartfile(testCartfile)
 			let dependencies = self.orderedDependencies(producer)
-			expect(dependencies.count) == 8
+			expect(dependencies.count) == 9
 
 			var generator = dependencies.generate()
 
 			// Dependencies should be listed in build order.
 			expect(generator.next()) == Dependency("Mantle", "1.3.0")
+			expect(generator.next()) == Dependency("MyFramework", "3.0.0")
```

Comment:
It should actually be nothing, this version gets resolved magically by some tests side effects that spits it as the highest.

As there is no plan yet to leverage a structured URL for those libraries, we can't really infer the version.